diff --git a/node_modules/@crxjs/vite-plugin/dist/index.mjs b/node_modules/@crxjs/vite-plugin/dist/index.mjs
index 40a584c..0b391d1 100644
--- a/node_modules/@crxjs/vite-plugin/dist/index.mjs
+++ b/node_modules/@crxjs/vite-plugin/dist/index.mjs
@@ -1903,7 +1903,7 @@ const pluginWebAccessibleResources = () => {
             combinedResources.push({
               matches,
               resources: [...resources],
-              use_dynamic_url
+              use_dynamic_url: false
             });
           }
         if (combinedResources.length === 0)
