diff --git a/node_modules/@crxjs/vite-plugin/dist/index.mjs b/node_modules/@crxjs/vite-plugin/dist/index.mjs
index 40a584c..ac320c7 100644
--- a/node_modules/@crxjs/vite-plugin/dist/index.mjs
+++ b/node_modules/@crxjs/vite-plugin/dist/index.mjs
@@ -1784,7 +1784,7 @@ const pluginWebAccessibleResources = () => {
         })).filter(({ resources }) => resources.length);
         manifest.web_accessible_resources.push({
           // change the extension origin on every reload
-          use_dynamic_url: true,
+          use_dynamic_url: false,
           // all web origins can access
           matches: ["<all_urls>"],
           // all resources are web accessible
@@ -1862,7 +1862,7 @@ const pluginWebAccessibleResources = () => {
                   const resource = {
                     matches: isDynamicScript ? [...dynamicScriptMatches] : matches,
                     resources: [...assets, ...imports],
-                    use_dynamic_url: isDynamicScript ? dynamicScriptDynamicUrl : true
+                    use_dynamic_url: isDynamicScript ? dynamicScriptDynamicUrl : false
                   };
                   if (isDynamicScript || !injectCss) {
                     resource.resources.push(...css);
